Sources/Xcodeproj/SchemesGenerator.swift

```diff
@@ -86,7 +86,7 @@ public final class SchemesGenerator {
             }
         })
         schemes.append(Scheme(
-            name: rootPackage.manifestName + "-Package", // TODO: use identity instead?
+            name: rootPackage.manifest.displayName + "-Package", // TODO: use identity instead?
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
This show up as the display name in Xcode, so we should keep using the display name.